server/schedulers/utils.go

```diff
@@ -151,7 +152,14 @@ func (p *balancePlan) getTolerantResource() int64 {
 }
 
 func adjustTolerantRatio(cluster opt.Cluster, kind core.ScheduleKind) float64 {
-	tolerantSizeRatio := cluster.GetOpts().GetTolerantSizeRatio()
+	var tolerantSizeRatio float64
+	switch cluster.GetClusterType() {
+	case core.RangeCluster:
+		// range cluster use a separate configuration
+		tolerantSizeRatio = cluster.(*schedule.RangeCluster).GetTolerantSizeRatio()
+	default:
+		tolerantSizeRatio = cluster.GetOpts().GetTolerantSizeRatio()
```

Comment:
How about adding GetTolerantSizeRatio in raft cluster?

Reply:
If so, we will need implementation for all cluster